devimint/src/devfed.rs

```diff
@@ -273,12 +273,14 @@ impl DevJitFed {
                     let gw_cln = gw_cln.get_try().await?.deref();
                     let gw_lnd = gw_lnd.get_try().await?.deref();
 
-                    open_channel_between_gateways(&bitcoind, gw_cln, gw_lnd).await?;
+                    let gateways: &[(&Gatewayd, &str)] =
+                        if let Some(gw_ldk) = gw_ldk.get_try().await?.deref() {
+                            &[(gw_cln, "CLN"), (gw_lnd, "LND"), (gw_ldk, "LDK")]
+                        } else {
+                            &[(gw_cln, "CLN"), (gw_lnd, "LND")]
+                        };
 
-                    if let Some(gw_ldk) = gw_ldk.get_try().await?.deref() {
-                        open_channel_between_gateways(&bitcoind, gw_ldk, gw_cln).await?;
-                        open_channel_between_gateways(&bitcoind, gw_ldk, gw_lnd).await?;
-                    }
+                    open_channels_between_gateways(&bitcoind, gateways).await?;
```

Comment:
It would probably be better to parallize N calls to open_channels_between_gateways with a different gateway pair each. Parallizing whole thing, instead of each step in the thing separately allows smoothing out differences in each step times.